static/css/styles.css

```diff
@@ -476,28 +476,44 @@ a:active {
     text-decoration: none;
 }
 
-p a {
+a {
     position: relative;
 }
 
-p a:after {
+a:after {
+    content: '';
+    position: absolute;
+    height: 0.2rem;
+    background-color: #00283C;
+    left: 0;
+    bottom: 0;
+    width: 0;
+    transition: all .3s;
+}
+a:hover:after {
+    width: 100%;
+}
+
+.article-page article p a:after {
```

Comment:
Why do you redo what you already have above?

Reply:
I don't understand what you're saying

Reply:
you're doing here `content: ''` (line 498) that is already default in `a` (line 484).
Same for `position:absolute`, `bottom: 0`, …

Reply:
oh yes right